src/index.js

```diff
@@ -1,6 +1,14 @@
+import { createElement } from 'react'
+import Widget from './widget'
+import { setup } from 'goober'
+import { shouldForwardProp } from 'goober/should-forward-prop'
```

Comment:
try to keep a convention of import order of:
- import from 3rd party packages, starts with staples like React
- import from "3rd party" but `@eqworks` scoped packages
- import from local context, order by distance (`.../` then `../` then `./`)